src/less/button/ds6/button-base.less

```diff
@@ -326,3 +332,24 @@ span.expand-btn__icon {
         margin: 0;
     }
 }
+
+button.expand-btn--primary,
+button.expand-btn--primary[disabled],
+button.expand-btn--primary[aria-disabled="true"] {
+    span.expand-btn__icon {
+        background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB3aWR0aD0iMjEuNiIgaGVpZ2h0PSIxMi41OCIgdmlld0JveD0iMS4zNSA1LjcgMjEuNiAxMi41OCI+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTEyLjE4NiAxOC4yODVjLS40NTEtLjAwOS0uODA5LS4xNjctMS4wNzUtLjQ0MWwtOS4zMzctOS42YTEuNTI3IDEuNTI3IDAgMCAxLS40MjQtLjk5OXYtLjEwOGMuMDE1LS4zODYuMTY2LS43NDEuNDI0LTEuMDA4LjU2LS41NzMgMS41MjktLjU3IDIuMDgyIDBsOC4yOTQgOC41MyA4LjI5Mi04LjUzMmMuNTU4LS41NyAxLjUyNi0uNTcgMi4wOCAwIC4yNjUuMjcuNDE2LjYyOS40MjggMS4wMXYuMDg3Yy0uMDEyLjM5MS0uMTY1Ljc1LS40MjcgMS4wMmwtOS4zMzMgOS42YTEuNDQzIDEuNDQzIDAgMCAxLTEuMDA0LjQ0MSIvPjwvc3ZnPg==');
```

Comment:
Is there a reason not to use a variable for the Base64 strings?

For readability sake, and to keep all of them in the same place, would be my argument in favor of moving them. Maybe move this to `/less/less/ds6/variables.less` in a variable named `@icon-chevron-down-btn-primary`, or something, then use that variable here.

Just thinking out loud.

Reply:
Yeah, I was waiting for us to settle on the naming convention (in the PR for #242) before extracting these out. We can definitely convert these to variables at some point.